### ISSUE TITLE
Inconsist color for new-tab button

### ISSUE THREAD

**Issue by poga** [beakerbrowser/beaker#220]
Operation System: OSX 10.12.1
Beaker Version: 0.5.1

This is kinda hard to describe, screenshot:
![2016-12-12 12 59 28](https://cloud.githubusercontent.com/assets/8631/21088572/1abcc086-c06b-11e6-8385-6f121a5343f8.png)

Zoom-in:
![untitled](https://cloud.githubusercontent.com/assets/8631/21088587/6954d814-c06b-11e6-8037-e2aa00c0fe1b.png)
The background color of the button is `#dddddd`, the trapezoid on the right is `#d8d8d8`

It can be fixed by removing this line: https://github.com/beakerbrowser/beaker/blob/master/app/stylesheets/shell-window/chrome-tabs.less#L51
```
      box-shadow: 1px -1px 1px rgba(0,0,0,.05); // this helps to darken the edge, which is oddly light from the 3d transform
```

However, I can't see the "oddly light" issue described here after I removed it. Maybe it's no longer needed?

**Comment by pfrazee** [comment]
Yeah it's still worth keeping (subtly improves the definition of the tabs' right borders) but it needs to be disabled on the add-tab btn. I'll fix it up in the next release, thanks for pointing it out

**Comment by poga** [comment]
FYI I can't reproduce this issue on current master 5a309b823ccd51181cbcd65a0eac28916ba7fe32.
